markdown/usage/introduction.md

```diff
@@ -80,6 +80,7 @@ Nextflow works best when you have an active internet connection, as it is able t
 * Use `-resume` to restart pipelines that did not complete. This ensures that successful tasks from the previous run wont be re-executed.
 * Use `nextflow log` to find names of all previous runs in your directory. These can be used with `-resume` to restart specific runs.
 * Be clever with multiple Nextflow configuration locations. For example, use `-profile` for your cluster configuration, `~/.nextflow/config` for your personal config such as `params.email` and a working directory `nextflow.config` file for reproducible run-specific configuration.
+* When you use `singularity` as your preferred container, we recommend to specify a cache directory with the [nextflow environment variable](https://www.nextflow.io/docs/latest/config.html#environment-variables) `NXF_SINGULARITY_CACHEDIR` in your `~./bash_profile` or `~/.bashrc` during the installation. This will store all your container images in one place, rather than repeatedly downloading it each time you run a pipeline. Just the base directory needs to be specified, Nextflow handles the folders and file names for you.
```

Comment:
https://github.com/nf-core/nf-co.re/pull/695#issuecomment-804036097

```suggestion
* If you use Singularity, we recommend that you specify a cache directory with the [nextflow environment variable](https://www.nextflow.io/docs/latest/config.html#environment-variables) `NXF_SINGULARITY_CACHEDIR` in your `~./bash_profile` or `~/.bashrc` during the installation. This will store all your container images in one place, rather than repeatedly downloading it each time you run a pipeline. Just the base directory needs to be specified, Nextflow handles the folders and file names for you.
```

@fabianegli - suggested changes in PR reviews are great for this kind of minor suggestion, means that they can be applied with one click :+1: